src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/custom.py

```diff
@@ -268,18 +269,23 @@ def _check_wincred(login_server):
                         with open(config_path, 'w') as output_file:
                             json.dump(content, output_file, indent=4)
                             output_file.close()
-                            return True
+                        return True
                     return False
                 except NoTTYException:
                     return False
             # Don't update config file or retry as this doesn't seem to be a wincred issue
             return False
         else:
+            import os
             content = {
                 "auths": {
                     login_server: {}
                 }
             }
+            try:
+                os.makedirs(docker_directory)
+            except FileExistsError:
```

Comment:
`FileExistsError` is only available in Python 3 so will fail in Python 2.
Maybe `OSError` is what you need?